run-make-check.sh

```diff
@@ -39,9 +43,16 @@ function run() {
     $DRY_RUN sudo /sbin/sysctl -q -w fs.aio-max-nr=$((65536 * $(nproc)))
 
     CHECK_MAKEOPTS=${CHECK_MAKEOPTS:-$DEFAULT_MAKEOPTS}
-    if ! $DRY_RUN ctest $CHECK_MAKEOPTS --output-on-failure; then
-        rm -fr ${TMPDIR:-/tmp}/ceph-asok.*
-        return 1
+    if in_jenkins; then
+        if ! ctest $CHECK_MAKEOPTS --no-compress-output --output-on-failure -T Test; then
+            # do not return failure, as the jenkins publisher will take care of this
+            rm -fr ${TMPDIR:-/tmp}/ceph-asok.*
```

Comment:
Let's see how this works... Last time I tried, when no results were found the executor threw pretty ugly Java exceptions :/

Reply:
@epuertat it works. see https://jenkins.ceph.com/job/ceph-pull-requests/72143/testReport/

Reply:
Great. I really like the [OpenBlueOcean view]( https://jenkins.ceph.com/blue/organizations/jenkins/ceph-pull-requests/detail/ceph-pull-requests/72143/tests).